src/main/webapp/app/shared/pipes/artemis-duration-from-seconds.pipe.ts

```diff
@@ -1,5 +1,12 @@
 import { Pipe, PipeTransform } from '@angular/core';
 
+interface Duration {
```

Comment:
Isn't this rather a `type` than an interface?

Reply:
Yes you’re right. It groups values, not methods. Changed it in 1c38c11ec1d9e605f011a08438a9dff0a51cfed3.